src/coreclr/scripts/superpmi.py

```diff
@@ -2700,7 +2700,7 @@ def process_base_jit_path_arg(coreclr_args):
 
         if coreclr_args.base_git_hash is None:
             # We've got the current hash; figure out the baseline hash.
-            command = [ "git", "merge-base", current_hash, "main" ]
+            command = [ "git", "merge-base", current_hash, "origin/main" ]
```

Comment:
NOTE: jitrollingbuild.py uses `origin/main` too: https://github.com/dotnet/runtime/blob/main/src/coreclr/scripts/jitrollingbuild.py#L238

Reply:
However, it assumes `origin` is `dotnet/runtime` but it can be one's fork instead. I guess I need to get the name that corresponds to `dotnet/runtime` via `git remote -v` 🤔